### PR TITLE
MGMT-13752: Add signatures for existing triage tickets as well

### PR DESCRIPTION
**Pull request by osherdp**
Currently, the code at assisted-installer-deployment adds signatures and edit the description & fields only for newly created tickets.

We want to do the same for existing tickets for 2 main reasons:
* If the original job that created a ticket couldn't finish adding all signatures for that ticket (e.g. logs-collector server got down temporarily, we had a bug in one of the signatures that has been fixed, etc.) we want to give it another try in the next jobs.
* The plan for moving to Jira attachments relies on us creating tickets with the attachments and processing them further afterwards.

In addition, this changes some other things:

* Moving all triage-related logic to a directory ``tools/triage``.
* Instead of loading all issues and then matching them with the failure IDs, just search for each one individually.
* Remove now redundant code for normalizing failure IDs.
* Created a "openshift versions" signature, instead of setting it at ticket creation.
* Invoke the description signature (named ``FailureDescription``) just like any other signature (instead of invocation at ticket creation).

### CODE REVIEW

File: tools/utils.py [hunk 1]
```diff
@@ -10,15 +9,3 @@ def get_credentials_from_netrc(*, hostname, netrc_file=None):
         return None, None
 
     return credentials[0], credentials[2]
-
-
-def normalize_jira_title(s):
```
Comment (osherdp) [inline tools/utils.py:15]: I expect all of this to be irrelevant, now that all new issues are normalized (and we only look at last month's issues)

File: tools/triage/create_triage_tickets.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            logs_url,
+            issue.key,
+            only_specific_signatures=None,
+            dry_run_file=None,
+        )
+        close_custom_domain_user_ticket(jira_client, issue.key)
+        close_by_signature.run_using_json(args.filters_json, jira_client, [issue])
+
+    if not args.filters_json:
+        return
```
Comment (adriengentil) [inline tools/triage/create_triage_tickets.py:72]: leftover?

Comment (osherdp) [reply tools/triage/create_triage_tickets.py:72]: oops. right

**Review by adriengentil — APPROVED** [review]
Looks good!

**Comment by osherdp** [comment]
Thanks! Starting the testing phase in jenkins

**Comment by osherdp** [comment]
I might do more changes. I'll let you know once it's all playing nicely in jenkins

**Comment by osherdp** [comment]
@adriengentil it should be ready for another review

**Review by adriengentil — APPROVED** [review]

**Comment by osherdp** [comment]
/unhold
test in jenkins passed, so unholding

**Pull request merged**